File: PHP Excel Examples/CellsApiDemo/Cell/calculateCell.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\CalculationOptions;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "CalculateCell_1.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet5";
$cellName = "G6";
$options = new  CalculationOptions();
$options->setCalcStackSize(0);
$options->setPrecisionStrategy("");
$options->setIgnoreError(true);
$options->setRecursive(true);
$apiInstance->calculateCell($name, $sheetName, $cellName, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/cell/calculateCell.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi
from spirecloudexcel.models import CalculationOptions

appId = "your id"
appKey = "your key"
configuration = ExcelConfiguration(appId, appKey)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)

name = "CalculateCell_1.xlsx"
folder = "/Cells/Cell/"
storage = ""
sheetName = "Sheet1"
cellName = "G6"
options = CalculationOptions()
options.calc_stack_size = 0
options.precision_strategy = ""
options.ignore_error = True
options.recursive = True
api.calculate_cell(name, sheetName, cellName, folder=folder, storage=storage)